liquibase-service/src/main/resources/db/rdb_modern/routines/023-sp_d_interview_postprocessing-001.sql

```diff
@@ -0,0 +1,614 @@
+               interview_type_cd   AS IX_TYPE_CD,
+               interview_loc_cd    AS IX_LOCATION_CD,
+               local_id,
+               record_status_cd,
+               record_status_time,
+               ADD_TIME,
+               add_user_id,
+               last_chg_time,
+               last_chg_user_id,
+               version_ctrl_nbr,
+               IX_STATUS,
+               IX_TYPE,
+               IX_LOCATION,
+               IX_INTERVIEWEE_ROLE
+        INTO #INTERVIEW_INIT
+        FROM dbo.nrt_interview
+        WHERE interview_uid in (SELECT value FROM STRING_SPLIT(@interview_uids, ','));
+
+        if
+            @debug = 'true'
+            select @Proc_Step_Name as step, *
+            from #INTERVIEW_INIT;
+
+        SELECT @ROWCOUNT_NO = @@ROWCOUNT;
+
+        INSERT INTO [DBO].[JOB_FLOW_LOG]
+        (BATCH_ID, [DATAFLOW_NAME], [PACKAGE_NAME], [STATUS_TYPE], [STEP_NUMBER], [STEP_NAME], [ROW_COUNT])
+        VALUES (@BATCH_ID, 'D_INTERVIEW', 'D_INTERVIEW', 'START', @PROC_STEP_NO, @PROC_STEP_NAME, @ROWCOUNT_NO);
+
+        COMMIT TRANSACTION;
+
+
+        BEGIN TRANSACTION;
+
+        SET @PROC_STEP_NO = @PROC_STEP_NO + 1;
+        SET @PROC_STEP_NAME = ' GENERATING #INTERVIEW_ANSWERS';
+
+        SELECT interview_uid,
+               rdb_column_nm,
+               answer_val
+        INTO #INTERVIEW_ANSWERS
+        FROM dbo.nrt_interview_answer
+        WHERE interview_uid in (SELECT value FROM STRING_SPLIT(@interview_uids, ','));
+
+        if
+            @debug = 'true'
+            select @Proc_Step_Name as step, *
+            from #INTERVIEW_ANSWERS;
+
+        SELECT @ROWCOUNT_NO = @@ROWCOUNT;
+
+        INSERT INTO [DBO].[JOB_FLOW_LOG]
+        (BATCH_ID, [DATAFLOW_NAME], [PACKAGE_NAME], [STATUS_TYPE], [STEP_NUMBER], [STEP_NAME], [ROW_COUNT])
+        VALUES (@BATCH_ID, 'D_INTERVIEW', 'D_INTERVIEW', 'START', @PROC_STEP_NO, @PROC_STEP_NAME, @ROWCOUNT_NO);
+
+        COMMIT TRANSACTION;
+
+        BEGIN
+            TRANSACTION;
+        SET
+            @PROC_STEP_NO = @PROC_STEP_NO + 1;
+        SET
+            @PROC_STEP_NAME = ' GENERATING #L_INTERVIEW_N ';
+
+
+        IF
+            OBJECT_ID('#L_INTERVIEW_N', 'U') IS NOT NULL
+            drop table #L_INTERVIEW_N;
+
+
+        CREATE TABLE #L_INTERVIEW_N
+        (
+            [id]              [int] IDENTITY
+                (
+                1,
+                1
+                )                              NOT NULL,
+            [INTERVIEW_UID]   [numeric](20, 0) NULL,
+            [D_INTERVIEW_KEY] [numeric](18, 0) NULL
+        ) ON [PRIMARY];
+
+
+        insert into #L_INTERVIEW_N ([INTERVIEW_UID])
+        SELECT DISTINCT INTERVIEW_UID
+        FROM #INTERVIEW_INIT
+        EXCEPT
+        SELECT INTERVIEW_UID
+        FROM dbo.L_INTERVIEW lt;
+
+        UPDATE #L_INTERVIEW_N
+        SET D_INTERVIEW_KEY = id + coalesce((SELECT MAX(D_INTERVIEW_KEY) FROM dbo.L_INTERVIEW), 0)
```

Comment:
If L_INTERVIEW made more sense over a new nrt key table, we should add that to the release documentation as a baseline table from RDB. 👍